src/subscriptions/SubscriptionManager.cpp

```diff
@@ -246,6 +262,14 @@ SubscriptionManager::pubLedger(
         getLedgerPubMessage(lgrInfo, fees, ledgerRange, txnCount)));
 
     ledgerSubscribers_.publish(message);
+
+    if (not bookChangesSubscribers_.empty())
```

Comment:
I like using `not` and `and`, but we don't use this elsewhere in the codebase. Should we start using this now? Is this supported on most linux platforms?

Reply:
actually, i was not aware this was in C/C++ forever and thought this was a recent feature of the language.. but it looks like it was always there: https://en.cppreference.com/w/cpp/language/operator_alternative
i usually use `not` when it's more readable (and i remember to use it) - like in this case. I don't mind using it or not but i do think this is more readable this way.

Reply:
I think it's more readable as well, I just want to make sure its supported on all of the linux platforms. I feel like there was some issue with this not being supported on some platforms, but maybe I'm thinking of windows, which I don't think we need to support.

Reply:
if i read the link i sent correctly then it's part of c/c++ standard since forever ago.. in that sense it's not only supported by both windows and linux but also by just about any compiler in existence. I could be misunderstanding the reference but i'm pretty sure that's the case. Having that said, i refactored the code a bit and the `not` went away naturally so no worries until next time we use it :D